plugin.video.themoviedb.helper/resources/lib/downloader.py

```diff
@@ -0,0 +1,138 @@
+import os
+import xbmc
+import xbmcaddon
+import xbmcvfs
+import xbmcgui
+import requests
+import zipfile
+import resources.lib.utils as utils
+from io import BytesIO
+try:  # Python 3
+    from urllib.parse import urlparse
+except ImportError:  # Python 2
+    from urlparse import urlparse
+
+
+class Downloader(object):
+    def __init__(self, download_url=None, extract_to=None):
+        self.addon = xbmcaddon.Addon()
+        self.download_url = download_url
+        self.extract_to = xbmc.translatePath(extract_to)
```

Comment:
note: I'll probably move this to `xbmcvfs` at some point

Reply:
Makes sense to be under xbmcvfs